src/internal/components/configurationControls/ConfigSection.tsx

```diff
@@ -14,6 +14,7 @@ export interface IConfigSectionProps {
     id: string;
     valuePath?: string;
     canBeToggled?: boolean;
+    children?: React.ReactElement | React.ReactElement[];
```

Comment:
where do need this in our PR?

Reply:
without this, there are type errors anywhere ConfigSection is used
`Property 'children' does not exist on type 'IntrinsicAttributes & Pick<IConfigSubsectionProps & WrappedComponentProps<"intl">, "valuePath" | ... 6 more ... | "pushData"> & { ...; } & RefAttributes<...>'.ts(2322)
`

Reply:
removed